src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -31,32 +32,64 @@
  * </ul>
  *
  */
-public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation {
+public final class RMSMappingQuality extends InfoFieldAnnotation implements StandardAnnotation, ReducibleAnnotation {
```

Comment:
this behavior matches 3.5 HC when run with `-G StandardHC -G Standard` - this behavior changed from 3.4 and 3.5. HC now prints `RAW_MQ` and not `MQ` (oddly, the header in 3.5 contains `MQ` and `RAW_MQ`)